source/how-tos/debug/debug-apache.rst

```diff
@@ -0,0 +1,79 @@
+.. _debug-apache:
+
+How to Debug Apache
+===================
+
+..  tip::
+
+  While this page is helpful it is no replacement for `apache httpd's documentation`. If
+  you are an administrator responsible for Open OnDemand, you are now an administrator of
+  apache httpd.  As such, you should get comfortable with it as from time to time you will
+  have to troubleshoot it.
+
+
+Log locations
+-------------
+
+.. _restart-apache:
+
+Restart services
+----------------
+
+   .. tabs::
+
+      .. tab:: RHEL/CentOS 7
+
+        .. code-block:: sh
+
+          sudo systemctl try-restart httpd24-httpd httpd24-htcacheclean
```

Comment:
I wonder if we really need to tell people to restart the htcacheclean daemon. I've never restarted that at OSC, Puppet doesn't deal with that service.  It's been in documentation forever and I honestly don't know what purpose it serves.

Reply:
```suggestion
          sudo systemctl try-restart httpd24-httpd
```

Always happy to remove something we don't understand. And honestly, if the unit file needs to depend on another, it should be `PartOf=httpd24-httpd.service` right?

Reply:
Actually the unit files we deploy are drop-ins so they actually modify the behavior of `httpd24-httpd.service`.  Example:

```
[root@webdev02 ~]# systemctl cat httpd24-httpd
# /usr/lib/systemd/system/httpd24-httpd.service
[Unit]
Description=The Apache HTTP Server
After=network.target remote-fs.target nss-lookup.target

[Service]
Type=notify
EnvironmentFile=/opt/rh/httpd24/root/etc/sysconfig/httpd
ExecStart=/opt/rh/httpd24/root/usr/sbin/httpd-scl-wrapper $OPTIONS -DFOREGROUND
ExecReload=/opt/rh/httpd24/root/usr/sbin/httpd-scl-wrapper $OPTIONS -k graceful
ExecStop=/opt/rh/httpd24/root/usr/sbin/httpd-scl-wrapper $OPTIONS -k graceful-stop
# We want systemd to give httpd some time to finish gracefully, but still want
# it to kill httpd after TimeoutStopSec if something went wrong during the
# graceful stop. Normally, Systemd sends SIGTERM signal right after the
# ExecStop, which would kill httpd. We are sending useless SIGCONT here to give
# httpd time to finish.
KillSignal=SIGCONT
PrivateTmp=true

[Install]
WantedBy=multi-user.target

# /etc/systemd/system/httpd24-httpd.service.d/httpd24-httpd-override.conf
#### File managed by Puppet ####

[Service]
PrivateTmp=false
# /etc/systemd/system/httpd24-httpd.service.d/ood.conf
[Service]
KillSignal=SIGTERM
KillMode=process
PrivateTmp=false
```